lib/board.rb

```diff
@@ -20,4 +31,18 @@ def position_taken?(index)
   def mark_board(index, mark = 'X')
     @board[index] = mark
   end
+
+  def full_board?
```

Comment:
I asked about this in the previous pr, but do you think it's redundant to use the `board` suffix on methods defined in the `board` class? When these are used in your program, you'll see things like `board.mark_board`, `board.full_board?`, etc. Do you prefer that to `board.mark`, and `board.full?`

Reply:
I'm trying to be super explanatory with the method names, but I guess that it should be assumed that if you're in the board class that it's about the board.